bin/src/run/loadConfigFile.ts

```diff
@@ -4,7 +4,8 @@ import * as rollup from 'rollup';
 import batchWarnings from './batchWarnings';
 import relativeId from '../../../src/utils/relativeId';
 import { handleError, stderr } from '../logging';
-import { InputOptions, OutputChunk } from '../../../src/rollup/index';
+import { InputOptions } from '../../../src/rollup/index';
+import { Bundle } from '../../../src/rollup';
```

Comment:
rollup/index?

Reply:
I guess the `/index` is unnecessary due to some TypeScript weirdness (which mirrors a node weirdness) 😉

Reply:
Ok, it seems removing the `/index` does not really play nice without our setup so I'll leave it.